server/logger.go

```diff
@@ -13,10 +14,31 @@ import (
 	"github.com/theplant/appkit/logtracing"
 )
 
+const traceHeaderKey = "traceparent"
+
 // Will absorb panics in earlier Middleware. Times the request and logs the result.
 func LogRequest(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(rw http.ResponseWriter, r *http.Request) {
-		ctx, span := logtracing.StartSpan(r.Context(), fmt.Sprintf("%s %s", r.Method, r.URL.Path))
+		var opts []logtracing.StartOption
+		trace := r.Header.Get(traceHeaderKey)
+		if len(trace) > 0 {
+			ts := strings.Split(trace, "-")
```

Comment:
What values are stored in this trace? What does each one represent?

Reply:
https://www.w3.org/TR/trace-context/#traceparent-header
The traceparent HTTP header field identifies the incoming request in a tracing system. It has four fields:
version
trace-id
parent-id
trace-flag